packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1369,9 +1369,8 @@ export const assertValidVersions = async (
 
   if (forkInstalled === 'false') {
     throw new Error(
-      `Detected invalid Foundry version. Please use Sphinx's fork of Foundry by\n` +
-        `running the command:\n` +
-        `foundryup --repo sphinx-labs/foundry --branch sphinx-patch-v0.1.0`
+      `Detected invalid Foundry version. Please update to the latest version of Foundry by running:\n` +
+        `foundryup`
```

Comment:
We still do this check since our changes were implemented in Foundry relatively recently so it's possible that the user may have not updated their installation in a while.

Reply:
Can you update the docs in the `assertValidVersions` function to reflect this?